src/content/release/breaking-changes/deprecate-focusable.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Deprecate `SemanticsProperties.focusable` and `SemanticsConfiguration.isFocusable`
+description: >
+  The `focusable` parameter has been replaced by `isFocused`.
+---
+
+## Summary
+
+The `SemanticsProperties.focusable` and `SemanticsConfiguration.isFocusable`
+parameters were deprecated in favor of the `SemanticsProperties.focused` and
+`SemanticsConfiguration.isFocused` parameters.
+
+## Context
+
+The `SemanticsConfiguration.isFocusable` is a boolean to describe
+whether the semantics node can have input focus, and
+`SemanticsConfiguration.isFocused` is a boolean to describe if the
+semantics node has input focus.
+
+The fix is the same with `SemanticsProperties.focusable` and `SemanticsProperties.focused`.
```

Comment:
This section should talk about why this we decide to deprecate isFocusable

Reply:
done